test/Test/Db.hs

```diff
@@ -95,7 +100,7 @@ upsertIssuesSpec env = describe "upsertIssues" $ do
     it "should leave the issues table unaffected when there's no corresponding repo" $
         env & issuesUnaffected (upsertIssues [invalidIssue]) `equals` True 
     it "should update the issue if the same issue already exists" $
-        env & issuesRowDifference (upsertIssues [updatedValidIssue]) `equals` [updatedValidIssue]
+        env & issuesRowDifference (upsertIssues [updatedValidIssue]) `equals` zipWith tagWithId [1..] [updatedValidIssue]
```

Comment:
I think that `tagWithId` function is useful But currently our tests are pretty small. And using `zipWith` here looks like an overcomplication. Having the following looks simpler to me. Don't forget, we still need to understand code written in tets :slightly_smiling_face:  

```haskell
[Id 1 `Withid` updatedValidIssue]
```